alyx/electrophysiology/models.py

```diff
@@ -106,9 +106,9 @@ class RecordingSite(BaseBrainLocation):
     site_no = models.IntegerField(help_text="which site on the probe")
 
 
-class ExtracellularRecording(TimeSeries):
+class ExtracellularRecording(Dataset):
```

Comment:
These changes are fine - but in the medium term we will need to think more about how implement tables that wrap particular dataset types.

* Do we even need them? (I think yes because they often store metadata that we want to be searchable)
* How do we populate them automatically?
* Can we have a constraint that they link to the a Dataset of the correct DatasetType?